### ISSUE TITLE
Colorize specs output

### ISSUE THREAD

**Issue by GoogleCodeExporter** [etorreborre/specs#64]
```
Hi, Eric,

I'd like to have an option to output tests in color. This is a patch.

http://colors.mx1.ru/~stepancheg/specs-color.diff

No tests, sorry.
```

Original issue reported on code.google.com by `stepan.k...@gmail.com` on 5 Mar 2009 at 9:04

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 5 Mar 2009 at 11:47
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 5 Mar 2009 at 11:48
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Thanks a lot for this patch Stepan. It is now available in 1.4.4-SNAPSHOT.
```

Original comment by `etorrebo...@gmail.com` on 6 Mar 2009 at 2:13
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `etorrebo...@gmail.com` on 14 Feb 2010 at 2:43
- Changed state: **Verified**
